src/main/webapp/availability.js

```diff
@@ -12,22 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let mousedown = false;
+
 function onAvailabilityLoad() {
   const loginInfo = getLoginInfo();
   loginInfo.then(supplyLogoutLinkOrRedirectHome); 
   loginInfo.then(getUserOrRedirectRegistration);
   loadAvailabilityTable(availabilityTableDiv(), browserTimezoneOffset());
 }
 
+function markMouseDown(tile) {
```

Comment:
How is the tile passed to this function used?

Reply:
This shouldn't actually be taking in tile as a parameter.